src/Apps/Collect/Routes/Collection/index.tsx

```diff
@@ -82,84 +82,91 @@ export class CollectionApp extends Component<CollectionAppProps> {
         fallbackHeaderImage?.edges[0]?.node?.image?.resized.url)
 
     return (
-      <AppContainer>
-        <FrameWithRecentlyViewed>
-          <Title>{`${title} - For Sale on Artsy`}</Title>
-          <Meta name="description" content={metadataDescription} />
-          <Meta property="og:url" content={collectionHref} />
-          <Meta property="og:image" content={socialImage} />
-          <Meta property="og:description" content={metadataDescription} />
-          <Meta property="twitter:description" content={metadataDescription} />
-          <Link rel="canonical" href={collectionHref} />
-          <BreadCrumbList
-            items={[
-              { path: "/collections", name: "Collections" },
-              { path: `/collection/${slug}`, name: title },
-            ]}
+      <>
+        <Title>{`${title} - For Sale on Artsy`}</Title>
+        <Meta name="description" content={metadataDescription} />
+        <Meta property="og:url" content={collectionHref} />
+        <Meta property="og:image" content={socialImage} />
+        <Meta property="og:description" content={metadataDescription} />
+        <Meta property="twitter:description" content={metadataDescription} />
+        <Link rel="canonical" href={collectionHref} />
+        <BreadCrumbList
+          items={[
+            { path: "/collections", name: "Collections" },
+            { path: `/collection/${slug}`, name: title },
+          ]}
+        />
+        {artworksConnection && (
+          <SeoProductsForArtworks artworks={artworksConnection} />
+        )}
+        {artworksConnection && (
+          <SeoProductsForCollections
+            descending_artworks={descending_artworks}
+            ascending_artworks={ascending_artworks}
+            collectionDescription={description}
+            collectionURL={collectionHref}
+            collectionName={title}
```

Comment:
Preference — I like to move meta tags outside the app component into a fragment just so I *know* they aren't visual.